packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.js

```diff
@@ -118,4 +180,40 @@ export default class QuanticFacet extends LightningElement {
   showLess() {
     this.facet.showLessValues();
   }
+
+  clearSelections(){
+    this.facet.deselectAll();
+    this.clearInput();
+  }
+
+  toggleFacetVisibility(){
+    this.facetIcon = this.isBodyVisible ? "utility:add" : "utility:dash";
+    this.isBodyVisible = !this.isBodyVisible;
+  }
+
+  handleKeyUp(){
+    console.log(this.canShowMore);
+    if(this.isSearchComplete){
+      this.isFacetSearchActive = this.input.value !== '';
+      this.facet.facetSearch.updateText(this.input.value);
+      this.facet.facetSearch.search();
+    }
+  }
+
+  clearInput(){
+    this.input.value = '';
+    this.updateState();
+  }
+
+  highlightResult(result, query){
```

Comment:
Did you take the highlighting part from atomic?

Reply:
Yes I took this from `facet-search-utils` in atomic. They did use css for the bold text, I will change mine to do the same